### DISCUSSION TITLE
Channels are not working

### DISCUSSION THREAD

**Discussion by Irishguy97** [iptv-org/community#1100] [Q&A]
Hi there there are some channels that are not working channel 9 Sydney error 403 
Venezuela channels are not working parser exception . Is there anyway you can update playlist so I can update my playlist on tivimate 
Thank you I look forward to hearing from you

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#how-to-report-a-broken-stream
